scripts/update_feconf.py

```diff
@@ -67,6 +67,7 @@ def _apply_changes_to_feconf():
 
 
 def _update_feconf():
+    """Updates the `feconf.py` file after doing the prerequisite checks."""
```

Comment:
Sorry for missing this earlier. I think it will be better to change ` to '

Reply:
Done